GameDataParser/Parsers/TrophyParser.cs

```diff
@@ -59,6 +59,14 @@ protected override List<TrophyMetadata> Parse()
                     newTrophy.ConditionCodes = condition.Attributes["code"].Value;
                 }
 
+                XmlAttribute? subJobLevelNode = reward.Attributes["subJobLevel"];
+                int subJobLevel = 0;
+
+                if (subJobLevelNode != null)
+                {
+                    subJobLevel = int.Parse(subJobLevelNode.Value);
+                }
+
```

Comment:
I think its easier for you to
`int.TryParse(reward.Attributes["subJobLevel"], out newGrade.RewardSubJobLevel);`

after the constructor